scripts/boldUpgradeCommon.ts

```diff
@@ -94,9 +98,9 @@ export const validateConfig = async (
   l1Rpc: providers.Provider
 ) => {
   // check all the config.contracts exist
-  if ((await l1Rpc.getCode(config.contracts.excessStakeReceiver)).length <= 2) {
```

Comment:
why is this commented out? can it just be deleted if we dont need it?

Reply:
ya I think we can just delete